Directory.Build.props

```diff
@@ -9,16 +9,8 @@
     <Description>A declarative command line and XML configuration parser for F# applications.</Description>
     <Authors>Eirik Tsarpalis</Authors>
     <Copyright>2019</Copyright>
-    <PackageTags>F#, argument, parser</PackageTags>
-    <RepositoryUrl>https://github.com/fsprojects/Argu/</RepositoryUrl>
```

Comment:
Could you keep `RepositoryUrl` in there? fsdocs tool uses it.

Reply:
does it use it _directly_ ? The ReproducibleBuilds nuget's props machinery derives it (probably indirectly via sourcelink?)

Reply:
It does, it evaluates the project and uses that property value as a substitution parameter.

Reply:
Hm, then I would have expected the `DotNet.ReproducibleBuilds` in the `paket.references` to make that continue to happen (depending on whether 'evaluates the project' means an MSBuild processing or a basic XML parse)

Reply:
It means `MSBuild` processing and it is currently not there
![image](https://github.com/fsprojects/Argu/assets/2621499/0e351362-2d56-4391-b243-03ad09645ee6)

Can you please just add it back?

Reply:
Yes, absolutely; 2s...

Reply:
Done.

Reply:
Thanks!